docs/installation/docker.rst

```diff
@@ -9,6 +9,8 @@ Docker
 
 There are several ways of installing Firefly III using Docker, which will be detailed below. If you're new to Docker or are not sure how to use Docker please thread carefully.
 
+Firefly III requires Linux containers running in your machine.
```

Comment:
What's the point of mentioning this?

Reply:
I was running Windows containers and I didn't know why I was unable to run the images

Reply:
I think you're mixing up your terminology. A **host** can be anything. Windows, Linux, Mac, doesn't matter, as long as the processor architecture is the same. On a host you download **images** which you use to run **containers**.

Firefly III images can be run (= turned into containers) on Intel, AMD or ARM host.

Reply:
I'm not talking about the host @JC5, I'm talking aboot the containers.
You can have Windows or Linux containers, which you can switch on the GUI thing they have.
I was running Windows containers for a project and I had to change them again to Linux (which is the default) in order to install Firefly

Reply:
If in other system types, there is no choice in choosing Windows or Linux containers, we can specify that

Reply:
You can see it in this image:

<https://imgur.com/a/KCG5uDr>

Reply:
Yeah I've been reading up on that and it'll only lead to confusion. You discovered it, I'm sure others will as well.

Reply:
Ok, so this one I delete and the one below gets merged?

Reply:
Yes :)

Reply:
Ok, I will do it when I get home.
Best regards!